src/post/userId.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class UserId extends React.Component{
+    render(){
+        return(
+            <p>{this.props.userId}</p>
+        )
+    }
+}
```

Comment:
Need to add the code below for using other components
"export default UserId;"

Reply:
What a big mistake! 👎